.travis.yml

```diff
@@ -3,11 +3,12 @@ gemfile:
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 rvm:
-  - 2.2.8
-  - 2.3.6
-  - 2.4.3
   - 2.5.1
+  - 2.6.3
+services:
+  - postgresql
 before_install: gem install bundler -v 1.12.4
```

Comment:
Based on the PR description, should this be version 2 or is this line un-needed?